### DISCUSSION TITLE
Plan to ship Aspire.Hosting.AWS as stable?

### DISCUSSION THREAD

**Discussion by cdschneider** [microsoft/aspire#4353] [General]
I noticed that `Aspire.Hosting.AWS` is amongst a small list of packages that was not shipped as stable (https://github.com/dotnet/aspire/pull/3668) with the GA release of Aspire.  Is there a list of outstanding issues or items that are needing to be completed in order to get this package ready for a stable release?

**Comment by davidfowl** [comment]
I believe @normj is waiting for feedback and usage before moving it to GA

**Comment by normj** [comment]
As well as getting feedback on what is in package today I also want us to finish the open PR that adds CDK support. That is a big area we will want to get feedback on.

Hopefully the fact that package is marked as preview doesn't hold you back from trying it. We only expect minor changes to the API if any while we collect feedback. We just want to be careful taking the preview tag off because then making even small API breaking changes will much harder.

**Comment by cdschneider** [comment]
OK great, I wondered if the CDK changes (this PR, I assume: #2225) was one that you wanted included in a stable version.  I'll keep on eye on the state of those changes.

The package being marked as preview is not preventing us from using it, I am already actively adding Aspire to our projects in anticipation of a stable release happening sometime soon.  Thanks for the info! @davidfowl @normj
